src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresViewIT.java

```diff
@@ -49,35 +50,38 @@
  * Integration tests for writing to Postgres views.
  */
 @Category(IntegrationTest.class)
-public class PostgresViewIT {
+public class PostgresViewIT extends BaseConnectorIT  {
 
   private static Logger log = LoggerFactory.getLogger(PostgresViewIT.class);
```

Comment:
Same thing here w/r/t `final`

Reply:
I think we accidentally made the class final instead of this field?

Reply:
@aakashnshah ping :)

Reply:
Done